### PR TITLE
PORTALS-2269: Add consortia component (Programs) to cancer complexity.

### PR DESCRIPTION
**Pull request by jay-hodgson**
Dependent on SRC release >= 3.0.34

### CODE REVIEW

**Comment by jay-hodgson** [comment]
Also depends on the Consortia Table being OPEN_DATA (syn34267412)

File: src/configurations/cancercomplexity/synapseConfigs/consortiaHomePage.ts [hunk 1]
```diff
@@ -0,0 +1,19 @@
+import { ProgramsProps } from 'synapse-react-client/dist/containers/home_page/programs/Programs'
+
+export const consortiaHomePageConfig: ProgramsProps = {
+  entityId:'syn34267412',
```
Comment (nickgros) [inline src/configurations/cancercomplexity/synapseConfigs/consortiaHomePage.ts:4]: Do we/stakeholders plan to add a version number later?

File: src/portal-components/DetailsPage/DetailsPageTabs.tsx [hunk 2]
```diff
@@ -41,7 +41,7 @@ const DetailsPageTabs: React.FunctionComponent<DetailsPageTabsProps> = (
       <div className="tab-groups">
         {tabConfigs.map((tab, index) => {
           return (
-            <Tooltip title={tab.toolTip ?? ''} placement='top'>
+            <Tooltip key={tab.uriValue} title={tab.toolTip ?? ''} placement='top'>
             <NavLink
               to={`${urlWithTrailingSlash}${tab.uriValue + search}`}
               key={`detailPage-tab-${index}`}
```
Comment (nickgros) [inline src/portal-components/DetailsPage/DetailsPageTabs.tsx:47]: Think this key could be removed

**Review by nickgros — APPROVED** [review]

File: src/portal-components/DetailsPage/DetailsPageTabs.tsx (hunk 2 above)
Comment (jay-hodgson) [reply src/portal-components/DetailsPage/DetailsPageTabs.tsx:47]: Error reported during build (and in IDE)

File: src/configurations/cancercomplexity/synapseConfigs/consortiaHomePage.ts (hunk 1 above)
Comment (jay-hodgson) [reply src/configurations/cancercomplexity/synapseConfigs/consortiaHomePage.ts:4]: Good question, has not been needed by other portal (AD)

**Pull request merged**